kubernetes_test.go

```diff
@@ -961,6 +961,74 @@ func TestKubernetesClusters_Destroy(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestKubernetesClusters_DeleteDangerous(t *testing.T) {
+	setup()
+	defer teardown()
+
+	kubeSvc := client.Kubernetes
+
+	mux.HandleFunc("/v2/kubernetes/clusters/deadbeef-dead-4aa5-beef-deadbeef347d/destroy_with_associated_resources/dangerous", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodDelete)
+	})
+
+	_, err := kubeSvc.DeleteDangerous(ctx, "deadbeef-dead-4aa5-beef-deadbeef347d")
+	require.NoError(t, err)
+}
+
+func TestKubernetesClusters_DeleteSelective(t *testing.T) {
+	setup()
+	defer teardown()
+
+	kubeSvc := client.Kubernetes
+
+	deleteRequest := &KubernetesClusterDeleteSelectiveRequest{
+		Volumes:         []string{"2241"},
+		VolumeSnapshots: []string{"7258"},
+		LoadBalancers:   []string{"9873"},
+	}
+
+	expectedReqJSON := `{"volumes":["2241"],"volume_snapshots":["7258"],"load_balancers":["9873"]}
+`
```

Comment:
nit: should this `char` be on a separate line or can it go right after the ending curly brace?